test/models/song_test.rb

```diff
@@ -33,4 +33,16 @@ class SongTest < ActiveSupport::TestCase
     assert_equal(song.name, "A Name")
     assert_equal(song.artist, "A Band")
   end
+  
+  test "search returns correct records" do
+    results = Song.search('God')
+    correct_results = songs(:God_be_praised)
+    assert_equal results.first, correct_results
+    assert results.one?
+  end
+  
+  test "search returns no records if no results" do
+    results = Song.search('banana')
```

Comment:
Hmmm I can see banana being in a song. Haha who knows? How about something that really could never exist like "e211d319d9b52c9f9215656db9c33deb".